Makefile

```diff
@@ -2,7 +2,7 @@
 #Under BSD License
 #See clock.c for the license detail.
 
-SRC = ttyclock.c ttyclock.h
```

Comment:
why?

Reply:
Header files in the source file list cause errors when executing make on some platforms, including the Mac. Sources are supposed to produce object files.

Reply:
hum. is that necessary for this PR to merge? it seems completely unrelated, could you split it out and expand on why this is necessary in the first place?

Reply:
Will do